nesta/core/orms/orm_utils.py

```diff
@@ -341,6 +344,14 @@ def cast_as_sql_python_type(field, data):
     return _data
 
 
+def get_session(db_env, section, database, Base):
```

Comment:
This used to be a part of `_filter_out_duplicates`, but is now used in both `_filter_out_duplicates` and the new `merge_duplicates`. Clearly factoring it out also allows for unit testing as well, as an added benefit